tools/packgen/src/PackGen.cpp

```diff
@@ -92,29 +92,33 @@ int PackGen::RunPackGen(int argc, char *argv[]) {
     return 1;
   }
 
-  // Check CMake
-  result = generator.ExecCommand("cmake --version");
-  if (result.second) {
-    cerr << "packgen error: CMake was not found" << endl;
-    return 1;
-  }
+  // Avoid CMake calls when packs to be generated don't contain components and verbose mode is disabled
+  if (!generator.m_noComponents || generator.m_verbose) {
```

Comment:
I am unsure why CMake should be run if in verbose mode even if no components are described. Just assume the cmake command is not specified for the generation of this pack.
I seem to be missing something.

Reply:
The verbose mode prints CMake target information and we recommend to use this feature when starting developing new packs (initially without components):
https://github.com/Open-CMSIS-Pack/devtools/tree/main/tools/packgen/docs#hints-for-generating-packs
Preventing CMake calls would break this feature.
Feel free to propose better solutions.

Reply:
Understood. Let's leave it for now.